examples/05_Collections/02_Set.md

```diff
@@ -0,0 +1,34 @@
+# Set
+
+A [Set](https://kotlinlang.org/docs/reference/collections.html) is an unordered collection that does not support duplicates. Like Lists and Maps, you can create a mutable set with `mutableSetOf()`, an immutable set with `setOf()` and a readonly view of a mutable set with the [Set&lt;T&gt;](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-set/index.html) interface.  
```

Comment:
Here you should change _immutable_ to _read-only_ as well.
Replace the second sentence with:
"For creating sets, there are functions `setOf()` and `mutableSetOf()`.  A read-only view of a mutable set can be obtained by casting it to `Set`."